ledfx/__main__.py

```diff
@@ -251,15 +245,16 @@ def main():
                 )
         else:
             p.nice(15)
-
+    # Setup Sentry
```

Comment:
For me this is an example of redundant comments, the intent of the code is self evident from well worded function / or variable names and the comment adds nothing other than bloat. Noting I don't do anywhere near the code review I used to, but this was a dangerous thing.

I would much rather see good structure and naming than extra comments.